src/test/e2e/api/admin/feature-archive.e2e.test.ts

```diff
@@ -25,78 +19,42 @@ beforeAll(async () => {
             },
         },
     });
-    await app.services.featureToggleServiceV2.createFeatureToggle(
-        'default',
-        {
-            name: 'featureX',
-            description: 'the #1 feature',
-        },
-        'test',
-    );
-    await app.services.featureToggleServiceV2.createFeatureToggle(
-        'default',
-        {
-            name: 'featureY',
-            description: 'soon to be the #1 feature',
-        },
-        'test',
-    );
-    await app.services.featureToggleServiceV2.createFeatureToggle(
-        'default',
-        {
-            name: 'featureZ',
-            description: 'terrible feature',
-        },
-        'test',
-    );
-    await app.services.featureToggleServiceV2.createFeatureToggle(
-        'default',
-        {
-            name: 'featureArchivedX',
-            description: 'the #1 feature',
-        },
-        'test',
-    );
-    await app.services.featureToggleServiceV2.archiveToggle(
-        'featureArchivedX',
-        'test',
-    );
-    await app.services.featureToggleServiceV2.createFeatureToggle(
-        'default',
-        {
-            name: 'featureArchivedY',
-            description: 'soon to be the #1 feature',
-        },
-        'test',
-    );
-    await app.services.featureToggleServiceV2.archiveToggle(
-        'featureArchivedY',
-        'test',
-    );
-    await app.services.featureToggleServiceV2.createFeatureToggle(
-        'default',
-        {
-            name: 'featureArchivedZ',
-            description: 'terrible feature',
-        },
-        'test',
-    );
-    await app.services.featureToggleServiceV2.archiveToggle(
-        'featureArchivedZ',
-        'test',
-    );
-    await app.services.featureToggleServiceV2.createFeatureToggle(
-        'default',
-        {
-            name: 'feature.with.variants',
-            description: 'A feature toggle with variants',
-            variants: [
-                { name: 'control', weight: 50 },
-                { name: 'new', weight: 50 },
-            ],
-        },
-        'test',
-    );
+    await app.createFeature({
```

Comment:
Here we're changing from direct service calls to HTTP API calls. Arguably, we could use services in the setup of the test (or we can even load the desired state of the DB), but because all of our services are exposed through HTTP APIs it doesn't feel bad to do it this way.

Reply:
I would argue HTTP API calls feel more `e2e` than direct calls to service methods that skip the controller and any other logic before reaching the service.

Reply:
Indeed, but in the setup, you just want to get you application state to a known state as soon as possible, it doesn't matter much how you put it in that stage, but if we can do it with HTTP calls exercising all our code while doing so, I think it increases the coverage